doc/architecture/backend.md

```diff
@@ -0,0 +1,56 @@
+# Backend Architecture
+
+## Structure
+
+The backend is split using a 3-layer architecture with controller, service and persistence layer.  
+Controller and service layer can be found in their respective folders.  
+The persistence layer is currently part of `model/entity/`.
+
+### Controller
+
+The controller layer contains all endpoints of the application.  
+The actual routes are defined in `config/routes/` while the controller layer only contains the actual implementation of the endpoints.
```

Comment:
I used links for now. Although they probably won't work with mdbook, so I need to think about a soultion there.

Reply:
It would be enough if the link checker knows that these files exists. I created #321. Imho it is okay if you cannot click on them when reading the mdbook.